frontend/src/concepts/pipelines/apiHooks/mlmd/types.ts

```diff
@@ -1,7 +1,9 @@
-import { Context } from '~/third_party/mlmd';
+import { Context, ContextType } from '~/third_party/mlmd';
 
 export type MlmdContext = Context;
 
+export type MlmdContextType = ContextType;
+
```

Comment:
not really sure why this is happening?

Reply:
Why do we need this?

Reply:
This is trying to differentiate the Mlmd context from all the other context types we have. It's basically a renaming to make more sense in the code when using it.